README.md

```
@@ -71,12 +71,17 @@ Trabalho principalmente com **Backend, APIs, SaaS Multi-Tenant e Plataformas de
-
-
+<picture>
+  <source
+    srcset="https://github-readme-stats.vercel.app/api?username=keroldias123&show_icons=true&theme=dark"
+    media="(prefers-color-scheme: dark)"
+  />
+  <source
+    srcset="https://github-readme-stats.vercel.app/api?username=keroldias123&show_icons=true"
+    media="(prefers-color-scheme: light), (prefers-color-scheme: no-preference)"
+  />
+  <img src="https://github-readme-stats.vercel.app/api?username=anuraghazra&show_icons=true" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)